DEVX-318: build Docker image to include kafka-connect-datagen (#626)
confluentinc/cp-docker-images: examples/cp-all-in-one/README.md

![image](../confluent-logo-300-2.png)

# Overview

> This `docker-compose.yml` includes all the components of the Confluent Platform to demonstrate how they can be configured to interoperate.

Reviewer: ```suggestion
This `docker-compose.yml` includes all of the Confluent Platform components and shows how you can configure them to interoperate.
```

> For an example of how to use this Docker setup, refer to the Confluent Platform quickstart: https://docs.confluent.io/current/quickstart/index.html

Reviewer: ```suggestion
For an example of how to use this Docker setup, refer to the :ref:`quickstart`.
```

Author: This isn't docs, so the ref won't resolve
